R/adapter.r

```diff
@@ -151,9 +151,11 @@ construct_file_adapter <- function() {
     if ('.rds' == substring(opts$resource, nchar(opts$resource) - 3, nchar(opts$resource)))
       readRDS(opts$resource)
     else {
+      test_if_url <- function(str) grepl("^https?://", str)
+      filesource <- ifelse(test_if_url(opts$resource), getURL(opts$resource), opts$resource)
```

Comment:
https://github.com/kirillseva/titanic/blob/master/titanic.R#L6-L9

```url %>% getUrl %>% read.csv(text=.)``` is the combination that works for me